includes/admin/upgrades/upgrade-functions.php

```diff
@@ -892,3 +897,31 @@ function give_v187_upgrades(){
 		}
 	}
 }
+
+/**
+ * Update Capabilities for Give_Worker User Role.
+ *
+ * This upgrade routine will update access rights for Give_Worker User Role.
+ *
+ * @since      1.8.8
+ */
+function give_v188_upgrades() {
```

Comment:
@mehul0810 You are right. It will not work without an upgrade. Because roles and capabilities store in database which needs to update 👍